### PR TITLE
Th 948 hobby types

### PR DESCRIPTION
**Pull request by melniiv**
## Additional notes :spiral_notepad:

### CODE REVIEW

File: src/domain/eventSearch/utils.tsx [hunk 1]
```diff
@@ -99,6 +107,40 @@ export const getCourseCategoryOptions = (t: TFunction): CategoryOption[] => [
   },
 ];
 
+// todo: fix icons and list of options when defined
+export const getCourseHobbyTypeOptions = (t: TFunction): HobbyTypeOption[] => [
+  {
+    icon: <IconMovies />,
```
Comment (Sartonon) [inline src/domain/eventSearch/utils.tsx:113]: Hmm are these icons used anywhere? same for the categories? can't remember

**Review by Sartonon — COMMENTED** [review]
Looks very good. Should we add some kind of test to Search component?

**Comment by Sartonon** [comment]
Are browser tests working locally for you?

File: src/domain/eventSearch/utils.tsx (hunk 1 above)
Comment (melniiv) [reply src/domain/eventSearch/utils.tsx:113]: not defined yet, i will fix it with actual values. even the list is temporary

**Comment by melniiv** [comment]
tests are commented out for right now, as well as for categories. when we have real values, they will be rechecked. At least one browser test is failing for me locally, investigating.

**Comment by melniiv** [comment]
tests for search component added (for event categories, course categories and course hobby types)

File: package.json [hunk 2]
```diff
@@ -86,11 +86,11 @@
     "build:client": "react-scripts build",
     "build:server": "NODE_ENV=production webpack --progress --mode production --config config/webpack.server.config.js",
     "codegen": "graphql-codegen -r dotenv/config dotenv_config_path=.env.development.local  --config codegen.yml",
-    "test": "TZ=Europe/Helsinki react-scripts test --env=jest-environment-jsdom-sixteen",
+    "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
     "ci": "CI=true yarn test --verbose --coverage",
-    "test:staged": "yarn test --watchAll=false --findRelatedTests",
+    "test:staged": "yarn test --watchAll=false --findRelatedTests --runInBand --forceExit",
```
Comment (Sartonon) [inline package.json:91]: why is --forceExit needed?

File: src/domain/eventSearch/__tests__/Search.test.tsx [hunk 3]
```diff
@@ -206,3 +206,28 @@ test('should change search query after selecting start date and pressing submit
...
+
+  //multiple selection
+  userEvent.click(chooseCategoryButton);
+  userEvent.click(screen.getByRole('checkbox', { name: /musiikki/i }));
+  userEvent.click(screen.getByRole('checkbox', { name: /museot/i }));
+  userEvent.click(screen.getByRole('button', { name: /hae/i }));
+  expect(history.location.pathname).toBe(pathname);
+  expect(history.location.search).toBe(
+    '?categories=movie,music,museum&text=jazz'
+  );
```
Comment (Sartonon) [inline src/domain/eventSearch/__tests__/Search.test.tsx:232]: 👍

File: package.json (hunk 2 above)
Comment (melniiv) [reply package.json:91]: it was a fix to the TZ issue, but i cherry picked the Jaakkos branch, and this line was merged incorrectly.. i ll fix

**Review by Sartonon — APPROVED** [review]
Great!

**Pull request merged**